### DISCUSSION TITLE
Editing input fields or content on page

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1397] [Q&A] [answered]
Hi I was curious if we have any patterns for editing items. I included a sample from what we currently have in SendGrid which opens up a side modal. I also included a pattern from Twilio. Mostly curious if we have guidelines around a similar use case of having to edit something. 

https://user-images.githubusercontent.com/81593493/116473361-51daed80-a834-11eb-8ec0-f81523ed05e4.mov

https://user-images.githubusercontent.com/81593493/116473375-569fa180-a834-11eb-9420-b4fe22b20ea4.mov

**Comment by nkrantz** [comment, marked as answer]
Hey @wilfredomerced! We don't currently have an established Edit Object pattern but it would be great to [get started on a contribution](https://paste.twilio.design/patterns/contribute/) if you'd be interested in that. @ishah-twilio had considered working on this contribution at one point, so might be worth reaching out to her to talk about collaborating.
